### DISCUSSION TITLE
Understrap 1.0 Child Theme Copy Assets

### DISCUSSION THREAD

**Discussion by nikunp** [holger1411/understrap#1417] [Q&A] [answered]
in copy-assets.js how do I copy just one file?

**Comment by bacoords** [comment, marked as answer]
We're using Node's [File system](https://nodejs.org/api/fs.html) commands, so you should be able to use a function like [`fs.copyFile()`](https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_mode_callback), which is essentially what we're using (we just wrap it in a custom function that runs it on each file inside a folder).
